modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/package.scala

```diff
@@ -90,25 +90,29 @@ package object engine {
 
   /**
     * Changes the `Status` and returns the new `Queue.State`.
-    *
-    * It also takes care of initiating the execution when transitioning to
-    * `Running` `Status`.
     */
   def switch(id: Sequence.Id)(st: SequenceState): HandleP[Unit] =
```

Comment:
`switch` was used for all sequence state changes. It add a special behavior for going to `Running`. Now that I added special ehavior for other state transitions, I replaced the use of `switch` for specialized functions `start`, `pause` and `cancelPause`.